soccer/gameplay/robocup-py.cpp

```diff
@@ -891,7 +891,6 @@ BOOST_PYTHON_MODULE(robocup) {
         .def_readonly("our_robots", &SystemState::self)
         .def_readonly("their_robots", &SystemState::opp)
         .def_readonly("ball", &SystemState::ball)
-        .def_readonly("game_state", &SystemState::gameState)
```

Comment:
Is this being used anywhere in python?

Reply:
No. The `main` module has a global gamestate, that's what is used.